docs/content/docs/flinkDev/building.md

```diff
@@ -79,6 +79,15 @@ The `fast` and `skip-webui-build` profiles have a significant impact on the buil
     # the version printed here must be 3.9, 3.10, 3.11 or 3.12
     ```
 
+3. Uv for building, installing and managing Python environments (optional)
+
+    The tool [uv](https://docs.astral.sh/uv/) is a useful Python package and environment manager that can make building and installing PyFlink from source much easier.
```

Comment:
the uv website calls it `An extremely fast Python package and project manager,` not `Python package and environment manager`.